ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift

```diff
@@ -360,7 +364,7 @@ scopedExample("`collect(count:)`") {
 }
 
 /*:
- ### ``collect(predicate:)` matching values inclusively`
+ ### `collect(predicate:) matching values inclusively`
```

Comment:
@ikesyo yes I was a little unsure about this one - the style of function description with a qualifier following seems to sit on its own in the playground.  Either way, 2292911 rectifies.